### ISSUE TITLE
Buckling a medical cyborg to a roller bed can produce infinite roller beds.

### ISSUE THREAD

**Issue by Chen-Yakumo** [Aurorastation/Aurora.3#10313]
Cyborg was outside on the asteroid stuck in a hole after falling down. I went to go get them and saw that there was no alternative route. Buckled them to a roller bed to see if I could do something with the wall before telling them to stay put while I went to get a hoist. Came back and there was two roller beds. I'm ASSUMING the stationbound deployed their roller bed. Stationbound was unable to resist out of the buckle. Closing the open roller bed did not close it, instead spawning a new roller bed, causing the stationbound to be nothing more than an infinite supply of roller beds.

**Comment by Playbahnosh** [comment]
I was the borg in question. Clarification, I was fully able to move, just couldn't get out of the hole. I was then buckled to the roller bed (I didn't even know borgs can be transported on roller beds). I tried to get up (resist out) from the roller bed. The bed lowered down, but I still wasn't able to move, so I tried to gather the roller bad with my roller bed tool. It also did not work, but it must've bugged out at that point, because I wasn't able to do _anything_ anymore. Couldn't move, couldn't use any tools, nothing, all I could do is chat. However, the roller bed I was on apparently kept spawning infinite roller beds.
